### PR TITLE
Added zygomys to Embeddable Scripting Languages

### PR DESCRIPTION
**Pull request by txgruppi**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Awesome Go [![Build Status](https://travis-ci.org/avelino/awesome-go.svg?branch=master)](https://travis-ci.org/avelino/awesome-go) [![Awesome](https://cdn.rawgit.com/sindresorhus/awesome/d7305f38d29fed78fa85652e3a63e154dd8e8829/media/badge.svg)](https://github.com/sindresorhus/awesome) [![Join the chat at https://gitter.im/avelino/awesome-go](https://badges.gitter.im/avelino/awesome-go.svg)](https://gitter.im/avelino/awesome-go?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
+e# Awesome Go [![Build Status](https://travis-ci.org/avelino/awesome-go.svg?branch=master)](https://travis-ci.org/avelino/awesome-go) [![Awesome](https://cdn.rawgit.com/sindresorhus/awesome/d7305f38d29fed78fa85652e3a63e154dd8e8829/media/badge.svg)](https://github.com/sindresorhus/awesome) [![Join the chat at https://gitter.im/avelino/awesome-go](https://badges.gitter.im/avelino/awesome-go.svg)](https://gitter.im/avelino/awesome-go?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
```
Comment (CentaurWarchief) [inline README.md]: There's an `e` lost here.

**Comment by matrixik** [comment]
Hello, thank you for contributing.
Looks very nice but have low `golint` score and there are also `go vet` errors:
https://goreportcard.com/report/github.com/glycerine/zygomys
Tests would also need some more love (unless black-box testing tests much more)
https://gocover.io/github.com/glycerine/zygomys/repl/
@glycerine

**Comment by glycerine** [comment]
@matrixik Thanks for your interest. Because zygo is a scripting language, the tests are almost all scripts that live in ./tests subdirectory, and are run by "make test".

**Comment by glycerine** [comment]
zygomys is in terrific shape, and is actively maintained and enhanced. The test suite is comprehensive, and the documentation extensive. The go vet minor notes have been addressed.

**Comment by matrixik** [comment]
Thank you @glycerine for fixing `go vet` errors.
If you omit `should have comment or be unexported` from `golint` output there are still some warnings that I think are worth fixing (but I still think there should be more docs).

**Comment by glycerine** [comment]
@matrixik That's an interesting opinion. Have you read the extensive wiki that describes the language? I really wish you would inform yourself before commenting ignorantly on my work.

**Comment by matrixik** [comment]
@glycerine That's great you create extensive wiki and I don't want to offend you in any way.
You put a lot of your valuable time into this package and I believe it's great fun for you  :smile: .

I'm just stating that `godoc` is lacking (close to the code docs, not wiki or other documentation).
Check here: https://godoc.org/github.com/glycerine/zygomys/repl
What would you feel if you want to use this great package (lets say you did not write it), would you be able to use it without checking what is happening in code for almost all functions?

Other `golint` warnings I mean like (they simply make code more readable, and more in line with other code in the wild - in similar way why we love `go fmt`):
> warning: should replace errors.New(fmt.Sprintf(...)) with fmt.Errorf(...)
> warning: error should be the last type when returning multiple items
> warning: don't use underscores in Go names; var completion_keywords should be completionKeywords
> warning: should omit 2nd value from range; this loop is equivalent to `for f := range ...` 
> warning: should omit type int from declaration of var p; it will be inferred from the right-hand side
> warning: error var SymNotFound should have name of the form ErrFoo
> warning: if block ends with a return statement, so drop this else and outdent its block
> warning: receiver name h should be consistent with previous receiver name hash for SexpHash
> warning: should replace precounts[name] += 1 with precounts[name]++
> warning: func JsonFunction should be JSONFunction
> warning: error strings should not end with punctuation

Again: I'm not attacking you, I just think that more uniform code decrease mental pressure for new developers when reading code that would like to help you with expanding this library (of course, I could be wrong).

Thank you for developing `zygomys`.

**Comment by txgruppi** [comment]
Guys.

I'm sorry for all this trouble. I forgot about the [Quality Standard](https://github.com/avelino/awesome-go/blob/master/CONTRIBUTING.md#quality-standard) when adding this link.

I just wanted to quickly add it since I'm always checking this repository before searching for packages.

I agree that some quality check is necessary and one of the things that makes Go great are the standards and tools for code quality.

@matrixik and @glycerine, sorry for partially ignoring the contribution guidelines. If I had followed it we would not have this "fight".

Let's just close this PR until `zygomys` is ok with the required code quality.

Cheers :beers:

**Comment by matrixik** [comment]
Ugh, or maybe I'm whining too much...

**Comment by txgruppi** [comment]
In my opinion you are right @matrixik

Some code quality is required to ensure this list remains as a source for good packages.

If you make an expection for one package how many other packages will ask for the same? It may be a problem.

Another good thing is that it forces the developer to make something "better" than usual, for most of us. I created another PR for a package where I didn't stop optimising and fixing it until I got a good coverage and A+

Cheers

**Pull request closed without merging**